test/pkg/test/framework.go

```diff
@@ -324,7 +320,25 @@ func (env *Framework) DeleteAllFrameworkTracedServiceNetworks(ctx aws.Context) {
 		}
 	}
 
-	//TODO: check and delete serviceNetwork VPC associations
+	var allServiceNetworkVpcAssociationIdsToBeDeleted []*string
```

Comment:
Let's `not` use test code to cleanup the resource for now.  Let developer debug why it is leaked and manually clean them up. 

Also, if these are not cleanup, the test should `stop`